src/Ajax_Inspector/Plugin.php

```diff
@@ -164,13 +164,12 @@ public function enqueue_ajax_button_script(): void {
 	 */
 	public function get_events_callback() {
 		// @todo Get events here using ORM and send response as JSON
-		if ( isset( $_POST['action'] ) ) {
-			$action = sanitize_text_field( $_POST['action'] );
+		if ( isset( $_GET['action'] ) ) {
 
-			$output = $action;
+            $events = tribe_events()->per_page(10)->page(1)->all();
 
 			// Return the result to the AJAX request
-			wp_send_json( $output );
+			wp_send_json( $events );
```

Comment:
`wp_send_json( [ 'events' => $events ] );` will give the same formatted output as the REST API.

Reply:
Let's make sure we don't mix tabs and spaces. Set that on your PHPStorm.

Reply:
For the PHPStorm tabs/spaces - I can see if I go to Preferences > Editor > Code Style > PHP I have a bunch of things selected in the 'Spaces' tab - is this where I need to change something?